lib/Models/Leaflet.ts

```diff
@@ -854,12 +851,15 @@ export default class Leaflet extends GlobeOrMap {
 
   getImageryLayersForItem(
     item: MappableMixin.MappableMixin
-  ): CesiumTileLayer[] {
+  ): (CesiumTileLayer | MapboxVectorCanvasTileLayer)[] {
```

Comment:
Maybe define a union type
```
type TerriaTileLayer = CesiumTileLayer | MapboxVectorCanvasTileLayer;
```
rather than repeating this type dec?

Reply:
there is only one usage atm - but in the future I think we should probs do this